sds_data_manager/lambda_code/efs_lambda/lambda_function.py

```diff
@@ -0,0 +1,133 @@
+import logging
+import os
+from pathlib import Path
+
+import boto3
+
+# Define the paths
+mount_path = os.environ.get("EFS_MOUNT_PATH")
+
+attitude_symlink_path = f"{mount_path}/latest_attitude_kernel.ah.a"
+ephemeris_symlink_path = f"{mount_path}/latest_ephemeris_kernel.bsp"
+
+
+def create_symlink(source_path: str, destination_path: str) -> None:
+    """Create a symlink from source_path to destination_path
+
+    Parameters
+    ----------
+    source_path : str
+        Source path of the symlink
+    destination_path : str
+        Destination path of the symlink
+    """
+
+    # Check if the symlink exists
+    if Path(destination_path).is_symlink():
+        # Remove the old symlink
+        Path(destination_path).unlink(missing_ok=True)
```

Comment:
You already know it exists with the if check. So, just go directly to removing the path, or take out the missing_ok I think.

```suggestion
    # Remove the old symlink
    Path(destination_path).unlink(missing_ok=True)
```